pyproject.toml

```diff
@@ -8,7 +8,6 @@ license = "Apache-2.0+GPL-2.0-only"
 [tool.poetry.dependencies]
 python = "^3.9"
 arcaflow-plugin-sdk = "0.9.0"
-ansible = "^6.5.0"
```

Comment:
The ansible module is still used in `metadata_plugin.py`. Please revert this change.